at_secondary/at_secondary_server/lib/src/connection/base_connection.dart

```diff
@@ -47,11 +47,23 @@ abstract class BaseConnection extends AtConnection {
       throw ConnectionInvalidException('Connection is invalid');
     }
     try {
+      logger.info('SENT: [${getMetaData().sessionID}] ${BaseConnection.truncateForLogging(data)}');
```

Comment:
@gkc this may cause lot of noise in prod since we log on every socket write. e.g image data will cause lot of encoded chars to be logged